Components/card/card.css

```diff
@@ -0,0 +1,402 @@
+*, *::before, *::after{
+    /* padding: 0;
+    margin: 0; */
+    box-sizing: border-box;
+  }
+  
+  
+  .container{
+    /* width: 100vw;
+    height: 100vh; */
+    display: flex;
+    justify-content: center;
+    align-items:center;
+    background: #5653e7;
+    position: relative;
+    margin: 2rem 2rem 2rem 3rem;
+    /* margin-left: 1rem; */
+    border-color: #000;
+  }
+
+  
+  .card{
+    width: 25rem;
+    border-radius: 1rem;
+    background: rgb(248, 246, 246);
+    /* box-shadow: 5px 5px 15px 15px rgba(#000, 0.15); */
+    box-shadow: 4px 4px 15px 15px  rgba(26, 35, 126, 0.2);
+
+    position : relative;
+    color: #434343;
+  }
+  
+  .card .card__container{
+    padding : 2rem;
+    width: 100%;
+    height: 100%;
+    background: white;
+    border-radius: 1rem;
+    border-color: black;
+    position: relative;
+  }
+  
+  .card .card__header{
+    margin-bottom: 1rem;
+    font-family: 'Playfair Display', serif;
+  }
+  
+  .card .card__body{
+    font-family: 'Roboto', sans-serif;
+  }
+  
+  .card::before{
+    position: absolute;
+    top:2rem;
+    right:-0.5rem;
+    content: '';
+    background: #283593;
+    height: 28px;
+    width: 28px;
+    transform : rotate(45deg);
+  }
+  
+  .card::after{
+    position: absolute;
+    content: attr(data-label);
+    top: 11px;
+    right: -14px;
+    padding: 0.5rem;
+    width: 10rem;
+    background: #3949ab;
+    color: white;
+    text-align: center;
+    font-family: 'Roboto', sans-serif;
+    box-shadow: 4px 4px 15px  rgba(26, 35, 126, 0.2);
+  }
+
+
+  /* dismiss */
+
+ 
+  .button {
```

Comment:
I think you should give the specific class to button because if you modify direct to button with there symetic  name it's specificity decreses and when anyone used your library in there  existing code base then user may have to face some issues